prac_07/project_management.py

```diff
@@ -0,0 +1,132 @@
+import datetime
+
+class Project:
+    def __init__(self, name, start_date, priority, cost_estimate, percent_complete):
```

Comment:
The Project class represents a project with attributes such as name, start date, priority, cost estimate, and percent complete. I